### PR TITLE
feat(onboarding): Change style of Name Screen

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

Part of the wider onboarding changes. [Figma](https://www.figma.com/file/NAp0jPTvldgl8AVajej3ns/Onboarding-Updates?node-id=1801%3A16171&t=AupXj9tOKyTBqY20-0)

Also removed lots of old experiment code. Restore flow now looks exactly the same as the new account flow

| Empty | With Dwight |
|:------:|:------:|
| Simulator Screen Shot - iPhone 13 - 2023-02-15 at 17 22 50 | Simulator Screen Shot - iPhone 13 - 2023-02-15 at 17 23 02

### Test plan

Unit tests updated

### Related issues

https://linear.app/valora/issue/ACT-603/update-name-screen

### CODE REVIEW

File: src/onboarding/registration/NameAndPicture.tsx [hunk 1]
```diff
@@ -1,154 +1,68 @@
...
 function NameAndPicture({ navigation, route }: Props) {
   const [nameInput, setNameInput] = useState('')
-  const [showSkipButton, nameType] = useMemo(getExperimentParams, [])
-  const { blockedAdjectives, blockedNouns } = useMemo(getBlockedUsernames, [])
   const cachedName = useTypedSelector(nameSelector)
-  const picture = useTypedSelector((state) => state.account.pictureUri)
   const choseToRestoreAccount = useTypedSelector((state) => state.account.choseToRestoreAccount)
   const recoveringFromStoreWipe = useTypedSelector(recoveringFromStoreWipeSelector)
   const { step, totalSteps } = useTypedSelector(registrationStepsSelector)
-  const shouldSkipProfilePicture = useTypedSelector((state) => state.app.skipProfilePicture)
```
Comment (satish-ravi) [inline src/onboarding/registration/NameAndPicture.tsx:95]: should we clean up this remote config flag from app state? Guess we could also make a single remote config flag cleanup once all dependencies on other flags (`showGuidedOnboarding`, `createAccountCopyTestType`) are removed.

Comment (jh2oman) [reply src/onboarding/registration/NameAndPicture.tsx:95]: I like the single pr idea!

File: locales/base/translation.json [hunk 2]
```diff
@@ -581,8 +581,10 @@
   "fullName": "Name",
   "fullNamePlaceholder": "First & Last name",
   "fullNameOrPseudonymPlaceholder": "First and last name or pseudonym",
```
[the remark this answers is not in the document]
Comment (jh2oman) [reply locales/base/translation.json:583]: yes we can

**Review by cajubelt — APPROVED** [review]
happy to see all the deletions

**Pull request merged**